src/textual/widgets/_tree.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         *,
         expanded: bool = True,
         allow_expand: bool = True,
-    ) -> None:
+    ):
```

Comment:
Accidental removal of return type hint here?

Reply:
😭  why are my fingers so fat?
Thanks for staying on top of things.